itou/companies/management/commands/import_ea_eatt.py

```diff
@@ -198,27 +198,19 @@ def process_file(self, file, *, wet_run=False):
         )
 
         # Display some "stats" about the dataset
-        self.stdout.write("-" * 80)
-        self.stdout.write(f"Rows in file: {info_stats['rows_in_file']}")
-        self.stdout.write(f"Rows after kind filter: {info_stats['rows_after_kind_filter']}")
-        self.stdout.write(f"Rows after deduplication: {info_stats['rows_after_deduplication']}")
-        self.stdout.write(f"Rows used: {len(ea_eatt_df)}")
-        self.stdout.write(f" > With a SIRET: {info_stats['rows_with_a_siret']}")
-        self.stdout.write(f" > With an empty email: {info_stats['rows_with_empty_email']}")
-        self.stdout.write(f" > Creatable: {info_stats['creatable_sirets']}")
-        self.stdout.write(f" >> Created: {info_stats['structures_created']}")
+        info_stats["rows_used"] = len(ea_eatt_df)
+        self.logger.info(
+            "EA/EATT import stats : %d structures created, %d structures updated, %d structures deleted",
```

Comment:
On a déjà le module `import_ea_eatt` de https://github.com/gip-inclusion/les-emplois/blob/03b30cb57a04e4b90f5c3809c15132b1c51a03ab/itou/utils/command.py#L63.
```suggestion
            "%d structures created, %d structures updated, %d structures deleted",
```